service/src/main/resources/application.yml

```diff
@@ -34,3 +34,5 @@ gpc-consumer:
   sds:
     url: ${GPC_CONSUMER_SDS_URL:}
     apiKey: ${GPC_CONSUMER_SDS_APIKEY:}
+    supplierOdsCode: ${GPC_SUPPLIER_ODS_CODE:}
```

Comment:
I wonder what behaviour the adaptor should have if the environment variable isn't provided.

I'm guessing at the moment the adaptor will try to fetch the ASID and throw a bad request exception.